src/main/java/seedu/address/model/property/Buyer.java

```diff
@@ -14,10 +15,14 @@
 import seedu.address.model.field.Price;
 import seedu.address.model.tag.Tag;
 
+/**
+ * Represents a Buyer in the address book.
+ */
 public class Buyer extends Person implements Listable, Taggable {
 
     private final Price maxPrice;
     private final Set<Tag> tags = new HashSet<>();
+    private final Set<Match> matches = new HashSet<>();
```

Comment:
In which scenario might we need this? I think since we already have an "edge list", would creating an "adjacency list" be redundant and potentially lead to inconsistencies?